common/source/docs/common-prearm-safety-checks.rst

```diff
@@ -253,9 +262,45 @@ Pre-arm checks that are failing will also be sent as messages to the GCS while d
+    Bad parameter: PSC_POSXY_P must be > 0                  Position controller misconfiguration                Increase specified parameter value to be above zero. See :ref:`Tuning Process Instructions<tuning-process-instructions>`
+    Check Q_ANGLE_MAX                                       Set above 80 degrees                                Reduce :ref:`Q_ANGLE_MAX<Q_ANGLE_MAX>` below 80; 30 degrees is typical
+    In landing sequence                                     Trying to arm while still in landing sequence       Reset mission;change to mission item not in a landing sequence
+    Invalid THR_FS_VALUE for reversed throttle input        THR_FS_VALUE pwm is not ABOVE the max throttle      Set :ref:`THR_FS_VALUE<THR_FS_VALUE>` above throttle maximum pwm
+    LIM_ROLL_CD too small x                                 Parameter set under 3 degrees                       Increase, 45 deg recommended minimum for adequate control
+    LIM_PITCH_MAX too small x                               Parameter set under 3 degrees                       Increase, 45 deg recommended minimum for adequate control
+    LIM_PITCH_MIN too large x                               Parameter set over 3 degrees                        Increase, 45 deg recommended minimum for adequate control
+    Mode not armable                                        Cannot arm from this mode                           Change Mode
+    Mode not QMODE                                          Q_OPTION set to prevent arming except in QMODE/AUTO Change Mode or reset :ref:`Q_OPTIONS<Q_OPTIONS>` bit 18
+    Motors: Check frame class and type                      Unknown or misconfigured frame class or type        Enter valid frame class and/or type
+    Motors: Check MOT_PWM_MIN and MOT_PWM_MAX               MOT_PWM_MIN or MOT_PWM_MAX misconfigured            Set :ref:`MOT_PWM_MIN<MOT_PWM_MIN>` = 1000 and :ref:`MOT_PWM_MAX<MOT_PWM_MAX>` = 2000 and repeat the :ref:`ESC calibration <esc-calibration>`
+    Motors: MOT_SPIN_ARM > MOT_SPIN_MIN                     MOT_SPIN_ARM is too high or MOT_SPIN_MIN is too low Reducse :ref:`MOT_SPIN_ARM<MOT_SPIN_ARM>` to below :ref:`MOT_SPIN_MIN<MOT_SPIN_MIN>`.  See :ref:`Seting motor range<set-motor-range>`
+    Motors: MOT_SPIN_MIN too high x > 0.3                   MOT_SPIN_MIN parameter value is too high            Reduce :ref:`MOT_SPIN_MIN<MOT_SPIN_MIN>` to below 0.3.  See :ref:`Setting motor range<set-motor-range>`
+    Motors: no SERVOx_FUNCTION set to MotorX                At least one motor output has not been configured   Check SERVOx_FUNCTION values for "Motor1", "Motor2", etc.  Check :ref:`ESC and motor configuration <connect-escs-and-motors>`
+    Q_ASSIST_SPEED is not set                               Q_ASSIST_SPEED has not been set                     Set :ref:`Q_ASSIST_SPEED<Q_ASSIST_SPEED>` = -1 for no Assistance, See :ref:`assisted_fixed_wing_flight`
```

Comment:
Should advise people to set it correctly rather than using the magic -1 to skip the check.

Reply:
for me -1 is correct, but I can just refer to the page (which now has a note about disabling QASSIT)